### ISSUE TITLE
deepin-music: Incorrect application version shown

### ISSUE THREAD

**Issue by felixonmars** [linuxdeepin/developer-center#3076]
Currently build date is shown in place of application version, this breaks reproducible build and is incorrect. Please consider using a real version number here.

![2020-05-02-122933-14c6af](https://user-images.githubusercontent.com/1006477/80855100-9909bf00-8c70-11ea-9b85-cdf0623bb131.png)

**Comment by BLumia** [comment]
Currently deepin-music should no longer show such incorrect version, but it won't show any version if `VERSION` is not defined. I suggest adding `-DVERSION="$pkgver"` at the build process.

https://github.com/archlinux/svntogit-community/blob/0c7727adae48f79e0511b69add9c3f07d32acb54/trunk/PKGBUILD#L28-L32

```
build() {
  cd deepin-music-$pkgver
  cmake -GNinja -DCMAKE_INSTALL_PREFIX=/usr -DVERSION="$pkgver" .
  ninja
}
```

![image](https://user-images.githubusercontent.com/10095765/165214244-56a3a773-cbca-47a3-8be9-5f893f61ea6b.png)

**Comment by felixonmars** [comment]
Fixed in deepin-music 6.2.15-2
